src/core/Akka.Persistence.Tests/Serialization/SerializerSpec.cs

```diff
@@ -97,267 +93,6 @@ public override int GetHashCode()
         }
     }
 
-    public class MyPayload2
-    {
-        public string Data { get; private set; }
-        public int N { get; private set; }
-
-        public MyPayload2(string data, int n)
-        {
-            Data = data;
-            N = n;
-        }
-
-        protected bool Equals(MyPayload2 other)
-        {
-            return string.Equals(Data, other.Data) && N == other.N;
-        }
-
-        public override bool Equals(object obj)
-        {
-            if (ReferenceEquals(null, obj)) return false;
-            if (ReferenceEquals(this, obj)) return true;
-            if (obj.GetType() != this.GetType()) return false;
-            return Equals((MyPayload2) obj);
-        }
-
-        public override int GetHashCode()
-        {
-            unchecked
-            {
-                return ((Data != null ? Data.GetHashCode() : 0)*397) ^ N;
-            }
-        }
-    }
-
-    public class MySnapshot
-    {
-        public string Data { get; private set; }
-
-        public MySnapshot(string data)
-        {
-            Data = data;
-        }
-
-        protected bool Equals(MySnapshot other)
-        {
-            return string.Equals(Data, other.Data);
-        }
-
-        public override bool Equals(object obj)
-        {
-            if (ReferenceEquals(null, obj)) return false;
-            if (ReferenceEquals(this, obj)) return true;
-            if (obj.GetType() != this.GetType()) return false;
-            return Equals((MySnapshot) obj);
-        }
-
-        public override int GetHashCode()
-        {
-            return (Data != null ? Data.GetHashCode() : 0);
-        }
-    }
-
-    public class MySnapshot2
-    {
-        public string Data { get; private set; }
-
-        public MySnapshot2(string data)
-        {
-            Data = data;
-        }
-
-        protected bool Equals(MySnapshot2 other)
-        {
-            return string.Equals(Data, other.Data);
-        }
-
-        public override bool Equals(object obj)
-        {
-            if (ReferenceEquals(null, obj)) return false;
-            if (ReferenceEquals(this, obj)) return true;
-            if (obj.GetType() != this.GetType()) return false;
-            return Equals((MySnapshot2) obj);
-        }
-
-        public override int GetHashCode()
-        {
-            return (Data != null ? Data.GetHashCode() : 0);
-        }
-    }
-
-    // this class was used when creating the data for the test
-    // "deserialize data when class is removed"
-    /*public class OldPayload
-    {
-        public char C { get; private set; }
-
-        public OldPayload(char c)
-        {
-            C = c;
-        }
-
-
-        public override string ToString()
-        {
-            return string.Format("OldPayload({0})", C);
-        }
-    }*/
-
-    public class SnapshotSerializerPersistenceSpec : AkkaSpec
```

Comment:
These tests were testing - how custom serializers serialize the persistence messages. I think it is redundant.